packages/core/src/fhir-to-cda/cda-templates/components/__tests__/encounter-examples.ts

```diff
@@ -0,0 +1,90 @@
+import { Encounter, Practitioner, Location } from "@medplum/fhirtypes";
+
+export const encounter1: Partial<Encounter> = {
+  status: "finished",
+  class: {
+    code: "99214",
+    display: "OV EST PT LEV 4",
+    system: "http://www.ama-assn.org/go/cpt",
+  },
+  type: [
+    {
+      text: "Urgent Care Office Visit",
+      coding: [
+        {
+          code: "99214",
```

Comment:
text and code for these should be in the xml right? 
something is wrong in the line 
`code: buildCodeCeFromCoding(encounter.resource.type)` in `createEntryFromEncounter`

Reply:
Oh.. Great catch! Will fix this!